src/Cake.Common/IO/GlobbingAliases.cs

```diff
@@ -41,6 +41,39 @@ public static FilePathCollection GetFiles(this ICakeContext context, string patt
         }
 
         /// <summary>
+        /// Gets all files matching the specified pattern.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="pattern">The glob pattern to match.</param>
+        /// <param name="predicate">The predicate used to filter files based on file system information.</param>
+        /// <returns>A <see cref="FilePathCollection" />.</returns>
+        /// <example>
```

Comment:
Ok will go thru and fix that and all existing example inconsistencies in that class.